src/components/Nav/Nav.js

```diff
@@ -1,7 +1,85 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import './Nav.scss';
 
 function Nav() {
-  return <div>Nav1</div>;
+  return (
+    <nav id="nav">
+      <div className="upperWrapper">
+        <div id="upper">
+          <div className="upperLeft">
+            <Link to="#">BOYVER</Link>
+            <Link to="#">보이쇼핑</Link>
+          </div>
+          <div className="upperRight">
+            <Link to="./user/signin">로그인</Link>
```

Comment:
```suggestion
            <Link to="/user/signin">로그인</Link>
```
* Link 컴포넌트에 to props는 URL에대한 정보를 받아오는 주소로, 절대경로로 입력합니다.
* 입력하신 './' 이 방법은, 파일을 찾을 때 입력하는 상대경로 입니다.
* 모든 Link컴포넌트에 to props에는 절대경로로 수정 해 주세요